modules/Core/includes/admin_widgets/discord.php

```diff
@@ -36,29 +36,13 @@
             } else {
                 // Validation errors
                 foreach ($validation->errors() as $validation_error) {
-                    if (strpos($validation_error, 'minimum') !== false) {
-                        // x must be a minimum of y characters long
-                        switch ($validation_error) {
-                            case (strpos($validation_error, 'discord_api_key') !== false):
-                                $errors[] = str_replace('discord_api_key', 'Discord Server ID', $validation_error);
-                                break;
-                        }
-                    } else if (strpos($validation_error, 'maximum') !== false) {
-                        // x must be a maximum of y characters long
-                        switch ($validation_error) {
-                            case (strpos($validation_error, 'discord_api_key') !== false):
-                                $errors[] = str_replace('discord_api_key', 'Discord Server ID', $validation_error);
-                                break;
-                        }
+                    if (strpos($validation_error, 'minimum') !== false || strpos($validation_error, 'maximum') !== false) {
+                        $errors[] = $language->get('admin', 'discord_id_length');
                     }
                     else if (strpos($validation_error, 'numeric') !== false) {
-                        // x must be a maximum of y characters long
-                        switch ($validation_error) {
-                            case (strpos($validation_error, 'discord_api_key') !== false):
-                                $errors[] = str_replace('discord_api_key', 'Discord Server ID', $validation_error);
-                                break;
-                        }
+                        $errors[] = $language->get('admin', 'discord_id_numeric');
                     }
+                    break;
```

Comment:
Shouldn't it show all errors, instead of breaking after showing the first one? Otherwise, what is the point of the for loop in the first place?

Reply:
Wow, how did I miss that.... Fixing